R/autoplot.curve_params.R

```diff
@@ -13,10 +13,13 @@
 #' library(dplyr)
 #' library(ggplot2)
 #'
-#' curve = "https://osf.io/download/rtw5k/" %>%
-#'   load_curve_params() %>%
+#' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
+#'   slice(1:1000)  %>% #reduce dataset for the purposes of this example
```

Comment:
- [ ] please add `.by = antigen_iso` to the `slice()` call, to avoid the possibility of getting only one antigen

Reply:
(it works currently, because the input RDS file is ordered by `iter` before `antigen_iso`, but that might change)